app/login/page.tsx

```diff
@@ -55,11 +55,11 @@ export default function LoginPage() {
   };
 
   return (
-    <div className={styles.login_container}>
+    <div className="login_container">

```

Comment:
Jeżeli używamy metodologii BEM to powinno to być z podwójnym podkreśleniem tj. __